src/components/NavItems.js

```diff
@@ -12,6 +17,12 @@ export function NavItems() {
     { navLinks: "About", menuLinks: [] },
     { navLinks: "Login", menuLinks: [] },
   ]
+
+  //adds favorite to nav if user is logged in
+  if (isLoggedIn) {
+    navItems.push({ navLinks: "Favorites", menuLinks: [] })
+  }
```

Comment:
Once we have the dropdown for the logout, profile, and favorite we can restructure this code.